core/src/main/java/com/novoda/noplayer/NoPlayerView.java

```diff
@@ -51,7 +51,7 @@ public View getContainerView() {
     }
 
     @Override
-    public SurfaceHolderRequester getSurfaceHolderRequester() {
+    public SurfaceRequester getSurfaceTextureRequester() {
```

Comment:
Do we need to leak `texture` here? SurfaceRequester should be enough?

Reply:
6312a81